src/Control/Comonad/Static.hs

```diff
@@ -0,0 +1,31 @@
+{-# LANGUAGE StaticPointers #-}
+
+module Control.Comonad.Static where
+
+import Control.Distributed.Closure
+import Data.Functor.Static
+import Data.Typeable (Typeable)
+
+-- | Instances of 'StaticExtend' should satisfy the following laws:
+--
+-- @
+-- 'staticExtend' f = 'staticMap' f . 'staticDuplicate'
+-- 'staticDuplicate' = 'staticExtend' (static 'id')
```

Comment:
I don't know why I wrote `staticMap` since it's obviously irrelevant from the two laws I cited after.
 
That being said, the two laws that you propose relate `staticExtend` and `staticDuplicate`, but don't constrain either beyond that. In contrast to either of the laws I pilfered from `semigroupoid`(which are associativity laws).

_again, I have no idea why I mentionned `staticMap`..._